sunpy/lightcurve/sources/eve.py

```diff
@@ -182,16 +183,16 @@ def _parse_level_0cs(fp):
         year = int(date_parts[0])
         month = int(date_parts[2])
         day = int(date_parts[3])
-        #last_pos = fp.tell()
-        #line = fp.readline()
-        #el = line.split()
-        #len
+        # last_pos = fp.tell()
+        # line = fp.readline()
+        # el = line.split()
+        # len
```

Comment:
Best to delete the commented out code.

Reply:
I agree, I only hesitated since I didn't write that code and I didn't know if it was being held on to for some reason.